content/docs/self-hosted-runners.md

```diff
@@ -202,16 +202,23 @@ env:
 
 ## Using on-premise machines as self-hosted runners
 
-You can also use the new `cml-runner` function to set up a local self-hosted
-runner. On your local machine or on-premise GPU cluster, you'll install CML as a
-package and then run:
+You can also use `cml-runner` to set up a local self-hosted runner on your local
+machine or on-premise GPU cluster.
 
 ```dvc
 cml-runner \
-  --repo $your_project_repository_url \
-  --token=$personal_access_token \
-  --labels tf \
-  --idle-timeout 180
+  --repo="$repository_url" \
+  --token="$personal_access_token" \
+  --labels="custom,runner" \
+  --idle-timeout=180
 ```
 
-Now your machine will be listening for workflows from your project repository.
+Now your machine will listen for jobs triggered on your repository and execute
+them locally.
+
+:warning: **Warning:** people with access to your repository (everybody for
```

Comment:
> `:warning:`

Does that render correctly? 😮 I didn't know that, I usually just paste the actual emoji (also works).

Reply:
Nope, that doesn't work 😏 

![image](https://user-images.githubusercontent.com/1477535/130274811-1582cdc8-8967-495e-8254-55b0f81ebd64.png)

Reply:
```suggestion
⚠️ **Warning:** people with access to your repository (everybody for
```

Reply:
I had to try! 😄  GitHub-flavoured Markdown usually accepts the colon notation. 🤷🏼‍♂️

Reply:
GH does render it tho 😆 

![image](https://user-images.githubusercontent.com/1477535/130275155-91d71c7a-0e72-47b7-a5d6-5c6289e5ccdf.png)